requirements.txt

```diff
@@ -1,5 +1,5 @@
-Django==1.11.27                         # BSD
-djangorestframework==3.9.4              # BSD
+Django==1.11                            # BSD
```

Comment:
Should the `Django` version be 2.x?

Reply:
I don't think we are dropping support for 1.11 just yet. Tbh though, this entry is kind of redundant because this file is used in the CI tests and the Django version installed is dynamic